katsdpcal/test/test_control.py

```diff
@@ -1079,6 +1091,86 @@ def prepare_heaps(self, rs, n_times,
         np.testing.assert_allclose(expected[1, 100], actual[1, 100], rtol=1e-4)
         np.testing.assert_allclose(expected, actual, rtol=1e-4)
 
+    async def wait_for_sensor(self, sensor, value, timeout):
+        """Wait `timeout` seconds for for `sensor` to have `value`."""
```

Comment:
Sorry, I must have missed pointing it out last time, but:
```suggestion
        """Wait `timeout` seconds for `sensor` to have `value`."""
```

Reply:
Whoops- that was my fault. Fixed.